airbyte-scheduler/app/src/main/java/io/airbyte/scheduler/app/SchedulerApp.java

```diff
@@ -273,7 +273,7 @@ public static void main(final String[] args) throws IOException, InterruptedExce
     final TemporalClient temporalClient = TemporalClient.production(temporalHost, workspaceRoot, configs);
 
     final Map<String, String> mdc = MDC.getCopyOfContextMap();
-    MetricSingleton.initializeMonitoringServiceDaemon("8082", mdc, configs.getPublishMetrics());
+    PrometheusMetricSingleton.initializeMonitoringServiceDaemon("8082", mdc, configs.getPublishMetrics());
```

Comment:
The title of this PR makes me think that it should also be moving all users of the prometheus singleton to use the dogStatsD singleton instead. Is the plan to do that in a separate PR?